README.md

```diff
@@ -39,16 +39,13 @@ can be created when the DOI is not yet calculated. Also in the title of the lice
 ARGUMENTS
 ---------
 
-    Usage: easy-license-creator <datasetID> <license-file>
-    Options:
+     -s, --sample    Indicates whether or not a sample license needs to be created
```

Comment:
Not sure what you mean. The usage is still (and even more complete) in the *synopsis* in the top of this page, right?